model-optimizer/mo/graph/perm_inputs.py

```diff
@@ -164,7 +164,7 @@ def transpose(op_node: Node, port_info: str, input_port: int):
     if len(permutation.perm) == 0:
         return
 
-    transpose_name = op_node.soft_get('name', op_node.id) + '/Transpose'
+    transpose_name = op_node.soft_get('name', op_node.id) + '/Transpose1'
```

Comment:
Is there a particular reason for this change?

Reply:
No